test/e2e/common/pods.go

```diff
@@ -848,4 +853,115 @@ var _ = framework.KubeDescribe("Pods", func() {
 		validatePodReadiness(false)
 
 	})
+
+	ginkgo.It("should run through the lifecycle of Pods and PodStatus", func() {
+		podResource := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods"}
+		testNs := "default"
+		testPodName := "pod-test"
+		testPodImage := "nginx"
+		testPodImage2 := "nginx"
```

Comment:
wrap the above strings in a `const (... )` block.
(but not `podResource` obviously)

Reply:
why not use a single `testPodImage` instead of adding the extra `testPodImage2`?

Reply:
That was a mistake, the images are in fact meant to be different